daemon/networkdriver/portallocator/portallocator.go

```diff
@@ -3,8 +3,23 @@ package portallocator
 import (
 	"errors"
 	"fmt"
+	"io/ioutil"
 	"net"
+	"regexp"
+	"strconv"
 	"sync"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	defaultPortRangeStart = 49153
+	defaultPortRangeEnd   = 65535
+)
+
+var (
+	BeginPortRange = 0
+	EndPortRange   = 0
```

Comment:
It feels a bit counterintuitive that these (mutable) `var`s are exported while the `const` above aren't.